python/docs/_static/copybutton.js

```diff
@@ -0,0 +1,66 @@
+// Copyright 2014 PSF. Licensed under the PYTHON SOFTWARE FOUNDATION LICENSE VERSION 2
```

Comment:
This license is compatible with ALv2. For completeness, I'd add an entry at the end of `LICENSE` noting the inclusion of software under this license, and add a file in `licenses/` containing the text of the license.

Reply:
@srowen Made the requested commits. Please check whether the LICENSE entries are appropriate.